pkg/scuzz/ata.go

```diff
@@ -0,0 +1,281 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package scuzz
+
+const (
+	oldSchoolBlockLen = 512
+)
+
+type (
+	// Cmd is an ATA command
```

Comment:
ATA command constant

can you also say where it's defined? source?

Reply:
let's hope this does itn :-)